src/main/deltachat/webxdc.ts

```diff
@@ -101,7 +101,15 @@ export default class DCWebxdc extends SplitOut {
           accounts_sessions.push(accountId)
           ses.protocol.registerBufferProtocol(
             'webxdc',
-            async (request, callback) => {
+            async (request, __callback) => {
```

Comment:
I don't really like the `__` prefix, I get that it means private in python, but I think right here it makes more sense to just call it callback or sth else.

Reply:
Just wanted to make sure that it's not used by mistake instead of `respond`.